### PR TITLE
Add scripts submodule

### PR DESCRIPTION
**Pull request by giorgiosironi**
Travis CI deals automatically with cloning the submodule as long as it is using a HTTPS remote and is public.

In terms of low overhead, this is minimal. Update is manual at the moment but standard (`git submodule update --remote`), with the possibility of updating automatically via [Dependabot](https://dependabot.com/submodules/).

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule ".scripts"]
+	path = .scripts
+	url = https://github.com/libero/scripts
```
Comment (giorgiosironi) [inline .gitmodules]: name can be changed if there's a better fit

Comment (thewilkybarkid) [reply .gitmodules]: Probably fine, not that specific but could cover multiple things (updates, testing, deployment etc). Unlikely to conflict too.

**Comment by thewilkybarkid** [comment]
Looks good, let's merge when there's something to use though.

**Comment by giorgiosironi** [comment]
Will update to include `travis/push-image.sh`

File: .travis.yml [hunk 2]
```diff
@@ -10,6 +10,7 @@ env:
         - IMAGE_TAG=$TRAVIS_COMMIT
 
 install:
+    - cat .scripts/README.md
```
Comment (thewilkybarkid) [inline .travis.yml]: Not needed anymore.

**Review by thewilkybarkid — APPROVED** [review]

**Comment by giorgiosironi** [comment]
Executed correctly (not on its main path):
```
0.01s $ .scripts/travis/push-image.sh dummy-api
not pushing an image, we are not on master
```

**Pull request merged**